docs/blog/2022-05-19-webops.md

```diff
@@ -0,0 +1,44 @@
+---
+description: Incremental static build capability is now available. Activate it to improve page loading time.
+tags: [webops]
+hide_table_of_contents: false
+---
+
+# Improvement in page loading time with Incremental Static Build
+
+Incremental Static Build means that only a fraction of the pages are generated during the website build. The rest are generated as shoppers access them.
+
+This means the first user to load a page, triggers its generation. After this first load, all shoppers who access that same page will experience optimized loading time.
+
+## What has changed?
+
+Now you can activate Incremental Static Build for your FastStore project. Part of the store's pages will be generated in each build, but the SKU pages will follow the Incremental Static Build behavior as described above. This tends to drastically improving their loading times overall.
+
+:::caution
+Keep in mind that:
+- Each deploy resets all pages generation status, which will then have to be requested by a user in order to be generated again.
+- Once a page is generated it will not be re-generated by external content changes. For example, if you edit a given product's description text in the VTEX platform, you must deploy a new website version in order for the page to be generated with the updated description text.
```

Comment:
```suggestion
- A deploy resets the caching status of all pages, meaning that pages previously cached must be requested again in order to be re-cached.
- Once a page is cached, it will not be re-generated by external content changes. For example, if you edit a given product's description in the VTEX platform, you must redeploy your website to generate the page with the updated description.
```